docs/installing-open-mpi/sysadmins.rst

```diff
@@ -0,0 +1,167 @@
+  case, we're excluding ``lo`` (localhost, because Open MPI has its own
+  internal loopback device) and ``eth0``.
+
+* *Tune the parameters for specific networks:* Each network plug-in
+  has a variety of different tunable parameters.  Use the ``ompi_info``
+  command to see what is available.  You show *all* available parameters
+  with:
+
+  .. code-block::
+
+      shell$ ompi_info --param all all
+
+  .. note:: Starting with Open MPI v1.8, ``ompi_info`` categorizes its
+     parameters in *levels*, as defined by the MPI_T interface from
+     the MPI standard.  You will need to specify ``--level 9`` (or
+     ``--all``) to show *all* MCA parameters.  `See this blog entry
+     <https://blogs.cisco.com/performance/open-mpi-and-the-mpi-3-mpi_t-interface>`_
+     for further information.
+
+  .. code-block:: sh
+
+     shell$ ompi_info --level 9
+     # or
+     shell$ ompi_info --all
+
+  Beware: there are *many* variables available.  You can limit the
+  output by showing all the parameters in a specific framework or in a
+  specific plug-in with the command line parameters:
+
+  .. code-block:: sh
+
+     shell$ ompi_info --param btl all --level 9
+
+  Shows all the parameters of all BTL components, and:
+
+  .. code-block:: sh
+
+     shell$ ompi_info --param btl tcp --level 9
+
+  Shows all the parameters of just the ``tcp`` BTL component.
+
+Adding a new plugin to a global Open MPI installation
+-----------------------------------------------------
+
+If you add a new component (such as support for a new network) to a global
+Open MPI installation, Open MPI will
+simply open the new plugin at run-time |mdash| your applications do not need
+to be recompiled or re-linked.
+
+Upgrading network hardware with a global Open MPI installation
+--------------------------------------------------------------
+
+If you upgrade your network hardware and your installation of Open MPI uses
+shared libraries, you simply need to
+recompile the Open MPI components that support that network and
+re-install them.
+
+More specifically, Open MPI shifts the dependency on the underlying
+network away from the MPI applications and to the Open MPI plug-ins.
+This is a major advantage over many other MPI implementations.
+MPI applications will simply open the new plugin when they run.
+
+User customization of a global Open MPI installation
+----------------------------------------------------
+
+It is typically sufficient for a single Open MPI
+installation (or perhaps a small number of Open MPI installations,
+depending on compiler interoperability,
+:ref:`see installing multiple copies of Open MPI
+<building-open-mpi-installation-location-multiple-copies-label>` for
+more information) to serve an entire parallel
+operating environment.
+
+However, a system-wide Open MPI installation can be customized on a
+per-user basis in two important ways:
+
+* *Per-user MCA parameters:* Each user can set their own set of MCA
```

Comment:
```suggestion
* *Per-user ref:`MCA parameters<label-running-setting-mca-param-values>`:* Each user can set their own set of MCA
```

Reply:
The 'ref' is missing a leading ':' in the suggestion, I'll correct this in another commit.